tests/robot-tests/tests/admin/bau/create_data_block_with_chart.robot

```diff
@@ -27,6 +30,22 @@ Upload subject
     ...    Academic Year 2025/26 (not Live)
     user uploads subject    UI test subject    upload-file-test.csv    upload-file-test.meta.csv
 
+Navigate to 'Footnotes' page
+    user waits for page to finish loading
+    user clicks link    Footnotes
+    user waits until h2 is visible    Footnotes
+
+Create footnote
+    user waits until page contains link    Create footnote
+    user clicks link    Create footnote
+    user waits until page does not contain loading spinner
+    user clicks footnote subject radio    UI test subject    Applies to all data
+    user clicks element    id:footnoteForm-content
```

Comment:
It isn't a different UI but I kept noticing transient exceptions when the tests tried to click the submit button. This selector seems to prevent this behavior